### ISSUE TITLE
Toast Disappears When Moving Mouse Over It

### ISSUE THREAD

**Issue by Li-Yuanyuan** [costlydeveloper/ngx-awesome-popup#19]
Thanks a lot for this library. It's really easy to use and works very well.

I found the "Toast" will disappear when moving mouse over it. It can be reproduced via the [demo](https://costlydeveloper.github.io/ngx-awesome-popup/#/playground/toast-generator) as well.

**Comment by Li-Yuanyuan** [comment]
One improvement (maybe):

Is it possible to make the font size as some configuration? Currently, it seems that I have to use both "::ng-deep" and "!important" to change the font size, eg:

```css
  ::ng-deep .text-wrapper {
    font-size: 12px !important;
  }
```

**Comment by costlydeveloper** [comment]
For sure that is the bug in the latest release, I will fix it soon. Thank you for reporting.

**Comment by costlydeveloper** [comment]
Fixed in [Release 2.2.0](https://github.com/costlydeveloper/ngx-awesome-popup/releases/tag/2.2.0)
